rachellegelden/Home-Together-Canada
# HTC-177: Added Home CSS

# [Issue 177](https://github.com/rachellegelden/Home-Together-Canada/issues/177)

closes #177 

## Summary
- Add CSS to home Page 
- Added new colours to `tailwind.config.js`
- Added base CSS to `tailwind.css`
- Added and imported new images to home page

## Relevant Motivation & Context
Make website look better!

## Testing Instructions
- Make sure you `npm install` on both client and root
- Run `npm start` in client
- Review CSS and the "feel" of the website

## Developer checklist prior to opening this pull request:

- [x] If CSS changes were introduced, change viewed in Chrome, Firefox and IE -- NOTE: NOT `IE`

## Reviewer
